designer/client/src/list/ListEdit.tsx

```diff
@@ -193,6 +193,44 @@ export function ListEdit() {
           }
           errorMessage={errors.title}
         />
+        <Radios
+          id="list-type"
+          name="type"
+          hint={{ children: i18n('list.typeHint') }}
+          fieldset={{
+            legend: {
+              className: 'govuk-fieldset__legend--s',
+              children: [i18n('list.type')]
+            }
+          }}
+          value={selectedList.type}
+          defaultValue="string"
```

Comment:
We always have `selectedList.type` (and it's updated on change) so we'd best remove `defaultValue`

```suggestion
```

Probably seen this in the browser console?

>Warning: Boolean contains an input of type radio with both checked and defaultChecked props. Input elements must be either controlled or uncontrolled (specify either the checked prop, or the defaultChecked prop, but not both)